src/Ares.php

```diff
@@ -1,46 +1,94 @@
 <?php
+/**
+ * This file is part of Ares.
+ * Copyright (c) 2015 Václav Vrbka (http://aurielle.cz)

```

Comment:
Despite this repository being a fork, almost all code was rewritten solely by myself. I have no problems giving credit where it's due, but is it really necessary to include h4kuna in the file headers? If so, what about the other files?